L1Trigger/VertexFinder/src/classes_def.xml

```diff
@@ -0,0 +1,33 @@
+<lcgdict>
+  <class name="l1tVertexFinder::InputData"/>
+  <class name="l1tVertexFinder::Vertex"/>
+  <class name="l1tVertexFinder::TP"/>
+  <class name="l1tVertexFinder::Stub"/>
+  <class name="std::map<DetId,DetId>"/>
+  <class name="edm::Ptr<l1tVertexFinder::Stub>"/>
+  <class name="edm::Ptr<l1tVertexFinder::TP>"/>
+  <class name="edm::RefToBase<TrackingParticle>"/>
+  <class name="edm::reftobase::BaseHolder<TrackingParticle>"/>
+  <class name="std::map<edm::Ptr<TrackingParticle>,edm::RefToBase<TrackingParticle> >"/>
```

Comment:
The `TrackingParticle` dictionaries should be defined in `SimDataFormats/TrackingAnalysis`.

Why is `RefToBase<TrackingParticle>` needed? I don't see anything inheriting from `TrackingParticle` even with this PR (unless I missed something).